### DISCUSSION TITLE
HIGH: Add merge queue-related events to Slack reminders integration

### DISCUSSION THREAD

**Discussion by purajit** [community/community#65225] [Pull Requests]
This is serious enough for us to avoid GHMQ on our largest repo, since it dramatically impacts developer productivity to not know when they may need to respond, when they can take action, when they can move on to the next step of their process, etc.

Slack reminders currently don't have merge queue-related events, which means it can be quite difficult to know when your PR fails a merge, or even if it succeeds in merging.

https://github.blog/2023-07-12-github-merge-queue-is-generally-available/ says
> If a merge fails, the pull request will be removed from the queue and a notification sent to the author of the pull request to check

However, the notifications list doesn't show many details besides "state change"
<img width="131" alt="image" src="https://github.com/community/community/assets/7026198/e7375fb9-1da9-44a6-adbe-443cb1fe7af0">
and I don't see any emails with any explicit "merge failed" - there's a notification about CI checks failing on the merge queue branch, but nothing else.

Interestingly, even with "Someone merges your pull request", the notification about the merge doesn't come through - I sense some special handling of how things work in the GitHub merge queue world
<img width="269" alt="image" src="https://github.com/community/community/assets/7026198/045ad38d-34e2-41a8-bd13-4a9d3b2855d1">

### Request
* Add notifications/state changes for successful merges
* Make notifications more explicit about merge queue status
* Make both these events available to the Slack reminder integration
* Add more events - when your PR is interrupted by a jump, etc

**Comment by dgmora** [comment]
As far as I've seen, this is still happening. Is this not planned or in the radar?

**Comment by quicksnap** [comment]
This would be extremely helpful. Would be nice to hear an update from GH on this.
